packages/ui/lib/components/Form/FieldLayout.tsx

```diff
@@ -72,6 +77,7 @@ export function FieldLayout(props: Props) {
       {label && (
         <label className={labelClass} htmlFor={label}>
           {label}
+          {required ? ' * ' : ''}
```

Comment:
not sure the browser will add the `*`; just have validation eventually

Reply:
> If you pass this to input, it will show * anyway. I don’t think we need to do this manually and let the browser figure out. You can style the star.

It's not happening right now... so until browsers do it. let's add a visual indication.

Reply:
actually, we not needed to do it for every input, we can just keep `required` from -> `inputProps.required`,

Reply:
also we need to add `required` also in the `input` field if we are not getting it from `inputProps `

Reply:
ha! let me try this!

Reply:
Good catch!